BridgeAppSDK/DataArchiving/SBADataArchive.m

```diff
@@ -163,6 +163,12 @@ - (void)insertDictionaryIntoArchive:(NSDictionary *)dictionary filename:(NSStrin
 
 - (void)insertDataIntoArchive :(NSData *)data filename: (NSString *)filename
 {
+    // Check that the file has not already been added
```

Comment:
For voice tests, it's probably not as straightforward but I think it would be better to include the last sound check file rather than the first, as that's going to be the one that corresponds to the time/place the voice test was finally recorded, in case researchers want to use it to try to compensate for the background noise in their analyses. I'm not sure this always applies in general to all possible file duplication scenarios going forward, though. Eventually we might want to be able to specify in a schema that there might be more than one of a given file, and use some naming convention when archiving them, so that researchers can see all the data and decide what to make of it, but for now this is the only situation I know about where we actually might care which of the "duplicate" files we save.